docs/example-session.md

```diff
@@ -29,28 +29,28 @@ Create an instance of 'Paratii' using default settings:
 
 We can now deploy the paratii contracts:
 
-    contracts = await paratii.deployAllContracts()
+    contracts = await paratii.getOrDeployContracts()
 
```

Comment:
I think it is better to change the code here, and not the docs.

Reply:
oh ok, I understand your point. I'll adjust it.